server/app/views/v1/grid_stats/stats.json.jbuilder

```diff
@@ -0,0 +1,15 @@
+json.from @node_stats[:from_time]
+json.to @node_stats[:to_time]
+json.stats @node_stats[:stats] do |stat|
+  json.timestamp stat[:timestamp]
+  json.data_points stat[:data_points]
```

Comment:
what `data_points` indicates? Just thinking the use case where you might need that.